packages/forms/src/documents/pdf/parsing-api.ts

```diff
@@ -108,17 +110,20 @@ const RadioGroup = z.object({
   component_type: z.literal('radio_group'),
   legend: z.string(),
   options: RadioGroupOption.array(),
+  page: z.number(),
```

Comment:
I'm pretty sure this is a github.com PR cache artifact, page has actually been removed from RadioGroupOption in the [pdf-pages branch](https://github.com/GSA-TTS/atj-platform/blob/be8da83940f06a6f690b328f7a9bcabc30ee8b22/packages/forms/src/documents/pdf/parsing-api.ts#L105)